common/thorhelper/thorcommon.cpp

```diff
@@ -1552,12 +1551,12 @@ class CRowStreamWriter : private IRowSerializerTarget, implements IExtRowWriter,
 
     void putRow(const void *row)
     {
-        _putRow(row, true);
+        _putRow(row);
     }
 
     void writeRow(const void *row)
     {
-        _putRow(row, false);
+        _putRow(row);
```

Comment:
This looks wrong - it will try and release the row.

Reply:
Yes, I mentioned "one bug I found" and this was it. I corrected it in the next PR but I will migrate that fix here.